packages/react-components/react-theme/src/alias/teamsDarkColor.ts

```diff
@@ -74,7 +75,8 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorNeutralBackground5Pressed: black, // #000000 Global.Color.Black
   colorNeutralBackground5Selected: grey[10], // #1a1a1a Global.Color.Grey.10
   colorNeutralBackground6: grey[20], // #333333 Global.Color.Grey.20
-  colorNeutralBackgroundInverted: grey[24], // #3d3d3d Global.Color.Grey.24
+  colorNeutralBackgroundStatic: grey[24], // #3d3d3d Global.Color.Grey.24
```

Comment:
```suggestion
  colorNeutralBackgroundStatic: white, // #ffffff Global.Color.White
```

Reply:
<img width="1518" alt="image" src="https://user-images.githubusercontent.com/9615899/183058638-510545d8-3c5b-464f-8ae3-0de3c1985fc7.png">

Reply:
The value in Figma conflicts with the guidance from the GH issue. I'm getting design input on which is the correct value.

Reply:
Confirmed with design that `grey 24` is the correct value here. The Figma should be updated to reflect that now.